src/graphql/datasources/spotify.js

```diff
@@ -26,6 +26,43 @@ class SpotifyAPI extends RESTDataSource {
       }
     );
   }
+
+  async createPlaylist(token, user_id, { name, description }) {
+    return await this.post(
+      `users/${user_id}/playlists/`,
+      { name, description },
+      {
+        headers: {
+          Authorization: 'Bearer ' + token,
```

Comment:
this is where the user is authenticated to ensure it really is the user.  I believe we briefly chatted about this last night :)

Reply:
Yes, that is correct.